workers/api-deployment/tasks.py

```
@@ -838,18 +838,33 @@ def _create_file_data(
+                    "rule_json": rule_data.get("rule_json"),
+                }
+                logger.info(
+                    f"API rules configured for workflow {workflow_id}: "
+                    f"percentage={manual_review_config['review_percentage']}, "
+                    f"rule_logic={manual_review_config['rule_logic']}"
+                )
+            else:
+                logger.info(f"No API rules configured for workflow {workflow_id}")
+        else:
+            logger.info(f"No API rules found for workflow {workflow_id}")
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)